static/docs/get-started/add-files.md

```diff
@@ -63,3 +63,28 @@ and if you check the `data.xml.dvc` meta-file you will see that it has this hash
 inside.
 
 </details>
+
+As an alternative, you can use `dvc import` instead of downloading and adding
```

Comment:
Need to move this under `### Expand to learn more about DVC internals`(and rename it into smth like `Expand to learn more`), since that "###" thing is used to name the `details` section.

Reply:
@efiop , Don't you think it's better to have `dvc import` right there?

Besides that, the `dvc import` paragraph also have some `expand to learn more`:
https://github.com/iterative/dvc.org/pull/128/files#diff-20b28dd90231b97c73562fdea43d8b0cR74

Reply:
Ah, good point. Didn't notice we already have <details> there. I'm afraid of overloading get-started with unnecessary information, that might only add to the confusion. Get-started should as simple as possible, after all. How about we use this text as a base for a new paragraph in  https://dvc.org/doc/user-guide/external-dependencies ? I think it would be much more suitable there, especially since we don't even mention `dvc import` there.

Reply:
@efiop, sorry for the delay, I implemented the requested changes, do you mind checking it again? :slightly_smiling_face:

Reply:
@mroutis Looks amazing! :tada: Thank you! :slightly_smiling_face: